src/fireblocks-sdk.ts

```diff
@@ -2081,6 +2088,78 @@ export class FireblocksSDK {
         return await this.getLinkedTokens({ status: TokenLinkStatus.PENDING, pageSize, pageCursor });
     }
 
+    /**
+     * Creates a new collection and links it
+     *
+     * @param {CreateCollectionRequest} payload - The payload containing information for collection creation
+     *
+     * @returns {CollectionLink} Response with created collection link ID
+     */
+    public async createNewCollection(payload: CreateCollectionRequest): Promise<CollectionLink> {
+        return await this.apiClient.issuePostRequest(`/v1/tokenization/collections`, payload);
+    }
+
+    /**
+     * Retrieves all linked collections in a paginated format.
+     *
+     * @param {Object} payload
+     * @param {TokenLinkStatus} payload.status - The status of linked collections (COMPLETED / PENDING). Default is COMPLETED
+     * @param {number} payload.pageSize - The number of results to return on the next page
+     * @param {string} payload.pageCursor - The cursor for the next page
+     *
+     * @returns {CollectionLink[]} A paginated array of linked collections
+     */
+    public async getLinkedCollections({ status, pageSize = DEFAULT_MAX_PAGE_SIZE, pageCursor }: GetTokenLinksFilter = {}): Promise<Web3PagedResponse<CollectionLink>> {
+        return await this.apiClient.issueGetRequest("/v1/tokenization/collections", {
+            status,
+            pageSize,
+            pageCursor,
+        });
+    }
+
+    /**
+     * Get a linked collection
+     *
+     * @returns CollectionLink
+     */
+    public async getLinkedCollection(id: string): Promise<CollectionLink> {
+        return await this.apiClient.issueGetRequest(`/v1/tokenization/collections/${id}`);
+    }
+
+    /**
+     * Unlink a collection
+     */
+    public async unlinkCollection(id: string): Promise<void> {
+        await this.apiClient.issueDeleteRequest(`/v1/tokenization/collections/${id}`);
+    }
+
+    /**
+     * Mint collection token
+     *
+     * @returns WriteCallFunctionResponseDto
+     */
+    public async mintCollectionToken(collectionId: string, payload: MintCollectionTokenRequest): Promise<WriteCallFunctionResponseDto> {
+        return await this.apiClient.issuePostRequest(`/v1/tokenization/collections/${collectionId}/tokens`, payload);
+    }
+
+    /**
+     * Get collection token details
+     *
+     * @returns CollectionTokenResponseDto
+     */
+    public async getLinkedCollectionTokenDetails(collectionId: string, tokenId: string): Promise<CollectionTokenResponseDto> {
+        return await this.apiClient.issueGetRequest(`/v1/tokenization/collections/${collectionId}/tokens/${tokenId}`);
+    }
+
+    /**
+     * Burn collection token
+     *
+     * @returns WriteCallFunctionResponseDto
+     */
+    public async burnCollectionToken(collectionId: string, payload: BurnCollectionTokenRequest): Promise<WriteCallFunctionResponseDto> {
+        return await this.apiClient.issuePostRequest(`/v1/tokenization/collections/${collectionId}/tokens`, payload);
```

Comment:
Isnt this delete request?

Reply:
Ignore the comment. Just read your comment indicating this will be removed!

Reply:
Burn removed.